lensless/utils/dataset.py

```diff
@@ -1508,6 +1514,26 @@ def __init__(
             )
             self.simulator = simulator
 
+        if bg_fp is not None and os.path.isfile(bg_fp):
```

Comment:
remove `and os.path.isfile(bg_fp)`. `load_image` below will check if the file exists, and the problem with this condition is that a user wouldn't get an error if the file doesn't exist